Blinking food

LarsTheHacker opened the pull request:
Added gold border and blinking animation to the food, making it easier for the user to find the food when it appears, and it appears like it's "spawning" in.

If you like, and it follows the rules, please accept for hacktoberfest2020. I'm not trying to spam, so if it seems like it, let me know, I'd be happy to make more changes to make this eligible. :)

LarsTheHacker committed:
Blinking food

Added gold border and blinking animation to the food, making it easier for the user to find the food when it appears, and it appears like it's "spawning" in.

If you like, and it follows the rules, please accept for hacktoberfest2020. I'm not trying to spam, so if it seems like it, let me know, I'd be happy to make more changes to make this eligible. :)

indremak commented:
I like your idea 🙂  I would suggest to create new animation, similar to existing animation, but it should change `background-color` to lighter. Because when `opacity` changes from 1 to 0 and then to 1 again, it doesn't look very good. If you change this, I'll merge your PR. 

When you create pull request there should be a description and name, that gives information about PR, this gives me more information about your changes 🙂

LarsTheHacker committed:
Food Spawn

Animation now gradually brings in the food, starting off with a lighter shade of the food. Does this over a period of 1 second, reaches full opacity. Also changed color of food to be a darker shade of magenta, offsetting it from the green field. Still has gold border.

LarsTheHacker committed:
Merge pull request #1 from LarsTheHacker/LarsTheHacker-patch-6

Food Spawn
